### DISCUSSION TITLE
Building a graphical launcher interface

### DISCUSSION THREAD

**Discussion by oisintheblue** [Windows81/Roblox-Freedom-Distribution#25] [Ideas]
A launcher ui application so you can more easily use this n stuff, but unlike the original rfd launcher, it would have more than 4 skintones, and overall better ui.

**Comment by Windows81** [comment]
Hey.  I don't intend to build a GUI for my project.  You're welcome to build that in a separate repo.

I want RFD to be as simple to use as possible.  That means having fewer moving parts for the user.  Building a GUI means longer build times and bulkier executables.  RFD should do two things well:

1. Host servers with a relevant `GameConfig.toml`, and
2. Join servers which are hosted from RFD.

**Comment by oisintheblue** [comment]
oh, aight. i understand.
i might aswell just wait for a fork of this to happen where it would have a launcher gui. also, is RFD gonna have only just 2018 and 2021? or will there eventually be a new client?

**Comment by Windows81** [comment]
No need to wait!  Perhaps making a GUI launcher can be your first foray into old-Rōblox stuff.

**Comment by Windows81** [comment]
I'm planning to add support for older CoreGui stuff for 2018M, so using older clients is not needed.

**Comment by orblua** [comment]
> I'm planning to add support for older CoreGui stuff for 2018M, so using older clients is not needed.

This would be great!
